integration-tests/bats/remotes.bats

```diff
@@ -1295,6 +1295,10 @@ SQL
     [ "$status" -eq 0 ]
     [[ "$output" =~ "test commit" ]] || false
 
+    # test pull with workspace up to date
+    run dolt pull
+    [ "$status" -eq 0 ]
```

Comment:
probably worth a quick check out `$output` here, too, since the bug prevented us from showing the right error message.